### PR TITLE
RHDEVDOCS-2550

### PR DESCRIPTION
**Pull request by rolfedh**
https://issues.redhat.com/browse/RHDEVDOCS-2550 (Removed image list for 4.6 release notes)
[enterprise-4.6]

### CODE REVIEW

**Comment by gabemontero** [comment]
LGTM @rolfedh

**Comment by rolfedh** [comment]
@xiuwang please ack. Thanks, Rolfe.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -1394,6 +1395,50 @@ The `OperatorSource` resource, part of the Marketplace API for the Operator Fram
...
+registry.redhat.io/ubi7/php-72
+registry.redhat.io/rhscl/php-72-rhel7
+registry.redhat.io/rhscl/postgresql-96-rhel7
+registry.redhat.io/rhscl/postgresql-96-rhel7
+registry.redhat.io/ubi7/python-36
+registry.redhat.io/rhscl/python-36-rhel7
+registry.access.redhat.com/ubi7/php-72
+registry.access.redhat.com/ubi7/python-36
+registry.redhat.io/dotnet/dotnet-31-rhel7:3.1
+registry.redhat.io/dotnet/dotnet-21-rhel7:2.1
```
Comment (xiuwang) [inline release_notes/ocp-4-6-release-notes.adoc:1416]: We don't remove imagestream for `registry.redhat.io/dotnet/dotnet-31-rhel7:3.1` and `registry.redhat.io/dotnet/dotnet-21-rhel7:2.1`
Details https://github.com/openshift/cluster-samples-operator/blob/release-4.6/assets/operator/ocp-x86_64/dotnet/imagestreams/dotnet-rhel.json

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -1394,6 +1395,50 @@ The `OperatorSource` resource, part of the Marketplace API for the Operator Fram
...
+registry.redhat.io/dotnet/dotnet-31-runtime-rhel7:3.1
+registry.redhat.io/dotnet/dotnet-21-runtime-rhel7:2.1
```
Comment (xiuwang) [inline release_notes/ocp-4-6-release-notes.adoc:1418]: We don't remove imagestreams for images `registry.redhat.io/dotnet/dotnet-31-runtime-rhel7:3.1` and `registry.redhat.io/dotnet/dotnet-21-runtime-rhel7:2.1`
https://github.com/openshift/cluster-samples-operator/blob/release-4.6/assets/operator/ocp-x86_64/dotnet/imagestreams/dotnet-runtime-rhel.json

File: release_notes/ocp-4-6-release-notes.adoc [hunk 3]
```diff
@@ -1394,6 +1395,50 @@ The `OperatorSource` resource, part of the Marketplace API for the Operator Fram
...
+The following images are no longer included in the samples imagestreams provided with {product-title}:
+
+----
+registry.redhat.io/rhscl/mysql-57-rhel7
+registry.redhat.io/rhscl/mongodb-34-rhel7
+registry.redhat.io/rhscl/nginx-114-rhel7
+registry.redhat.io/ubi7/php-72
+registry.redhat.io/rhscl/php-72-rhel7
+registry.redhat.io/rhscl/postgresql-96-rhel7
+registry.redhat.io/rhscl/postgresql-96-rhel7
```
Comment (xiuwang) [inline release_notes/ocp-4-6-release-notes.adoc:1410]: Duplicated image `registry.redhat.io/rhscl/postgresql-96-rhel7` with above image

File: release_notes/ocp-4-6-release-notes.adoc [hunk 4]
```diff
@@ -1394,6 +1395,50 @@ The `OperatorSource` resource, part of the Marketplace API for the Operator Fram
...
+registry.redhat.io/dotnet/dotnet-21-rhel7:2.1
+registry.redhat.io/dotnet/dotnet-31-runtime-rhel7:3.1
+registry.redhat.io/dotnet/dotnet-21-runtime-rhel7:2.1
+registry.redhat.io/fuse7/fuse-karaf-openshift:1.6
+registry.redhat.io/devtools/go-toolset-rhel7:1.11.5
+registry.redhat.io/rhscl/mariadb-102-rhel7
+registry.redhat.io/rhscl/nginx-114-rhel7
+registry.redhat.io/rhscl/php-72-rhel7
+registry.redhat.io/rhscl/postgresql-96-rhel7
+registry.redhat.io/rhscl/python-36-rhel7
```
Comment (xiuwang) [inline release_notes/ocp-4-6-release-notes.adoc:1425]: Dupilcated `registry.redhat.io/rhscl/nginx-114-rhel7` , `registry.redhat.io/rhscl/php-72-rhel7`, `registry.redhat.io/rhscl/postgresql-96-rhel7` and `registry.redhat.io/rhscl/python-36-rhel7`

File: release_notes/ocp-4-6-release-notes.adoc [hunk 5]
```diff
@@ -1394,6 +1395,50 @@ The `OperatorSource` resource, part of the Marketplace API for the Operator Fram
...
+registry.redhat.io/rhdm-7/rhdm-kieserver-rhel8:7.7.1
+registry.redhat.io/rhpam-7/rhpam-businesscentral-monitoring-rhel8:7.7.0
+registry.redhat.io/rhpam-7/rhpam-businesscentral-monitoring-rhel8:7.7.1
+registry.redhat.io/rhpam-7/rhpam-businesscentral-rhel8:7.7.0
+registry.redhat.io/rhpam-7/rhpam-businesscentral-rhel8:7.7.1
+registry.redhat.io/rhpam-7/rhpam-kieserver-rhel8:7.7.0
+registry.redhat.io/rhpam-7/rhpam-kieserver-rhel8:7.7.1
+registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.7.0
+registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.7.1
+registry.redhat.io/rhscl/ruby-27-rhel7
```
Comment (xiuwang) [inline release_notes/ocp-4-6-release-notes.adoc:1438]: We don't remove imagestream for image `registry.redhat.io/rhscl/ruby-27-rhel7`
https://github.com/openshift/cluster-samples-operator/blob/release-4.6/assets/operator/ocp-x86_64/ruby/imagestreams/ruby-rhel.json#L73

**Comment by ahardin-rh** [comment]
LGTM

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (rolfedh) [reply release_notes/ocp-4-6-release-notes.adoc:1416]: removed

File: release_notes/ocp-4-6-release-notes.adoc (hunk 2 above)
Comment (rolfedh) [reply release_notes/ocp-4-6-release-notes.adoc:1418]: removed

File: release_notes/ocp-4-6-release-notes.adoc (hunk 3 above)
Comment (rolfedh) [reply release_notes/ocp-4-6-release-notes.adoc:1410]: removed duplicate lines

File: release_notes/ocp-4-6-release-notes.adoc (hunk 4 above)
Comment (rolfedh) [reply release_notes/ocp-4-6-release-notes.adoc:1425]: removed

File: release_notes/ocp-4-6-release-notes.adoc (hunk 5 above)
Comment (rolfedh) [reply release_notes/ocp-4-6-release-notes.adoc:1438]: updated

**Comment by rolfedh** [comment]
@ahardin-rh Resolved all discussions. Ready to merge.

**Comment by xiuwang** [comment]
LGTM

**Pull request merged**